### PR TITLE
Namanb009 windowtitlefix

### PR DESCRIPTION
**Pull request by namanb009**
The window.SetWindowName() should always be called after the window.Render(), as mentioned in the VTK documentation.

### CODE REVIEW

File: fury/window.py [hunk 1]
```diff
@@ -494,7 +489,12 @@ def initialize(self):
 
     def render(self):
         """Render only once."""
-        self.window.Render()
+        self.window.Render() 
+        if self.title=="FURY":
```
Comment (skoudoro) [inline fury/window.py]: `self.title.upper()` instead

**Comment by namanb009** [comment]
Thank you @skoudoro 
I am definitely more than happy to contribute.

I've made the changes. Pushing them right away.

File: fury/window.py [hunk 2]
```diff
@@ -494,7 +489,12 @@ def initialize(self):
...
+            self.window.SetWindowName(self.title + " " + fury_version)
+        else:
+            self.window.SetWindowName(self.title)
+
```
Comment (namanb009) [inline fury/window.py]: Fixed PEP8 issues

**Comment by Garyfallidis** [comment]
Good catch @namanb009. Merging...

**Comment by namanb009** [comment]
Thank You @Garyfallidis Glad to make my first contribution.

**Pull request merged**